test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,172 @@
+local t = require('luatest')
+local fio = require('fio')
+local g = t.group('tarantool_role_test')
+local fiber = require('fiber')
+local fun   = require('fun')
+local net_box = require('net.box')
+
+local iter = fun.iter
+
+local Server = require('test.integration.server')
+
+local tmpdir = fio.tempdir()
+
+local function split(self, sep)
+    local fields = {}
+    local pattern = string.format("([^%s]+)", sep)
+    self:gsub(pattern, function(c) table.insert(fields, c) end)
+    return fields
+end
+
+local function reducer(res, l, r)
+    if res ~= nil then
+        return res
+    end
+    if tonumber(l) == tonumber(r) then
+        return nil
+    end
+    return tonumber(l) > tonumber(r)
+end
+
+--- e.g
+--- input: "2.11.0"
+--- output: {"2", "11", "0"}
+local function split_version(version_string)
+    local vtable  = split(version_string, '.')
+    local vtable2 = split(vtable[3],  '-')
+    vtable[3], vtable[4] = vtable2[1], vtable2[2]
+    return vtable
+end
+
+local function isTarantoolVersionGreater(expected, version)
+    version = version or _TARANTOOL
+    if type(version) == 'string' then
+        version = split_version(version)
+    end
+    local res = iter(version):zip(expected):reduce(reducer, nil)
+    if res or res == nil then res = true end
+    return res
+end
```

Comment:
https://github.com/tarantool/expirationd/blob/8c669678dde5f1c88e862dacb33a8e79f4ddd9a3/test/helper.lua#L250-L259

Please, use a code from helper.lua and move a supported version method there.